components/tag/suggestion/suggestion.ts

```diff
@@ -0,0 +1,78 @@
+import View from '!view!./suggestion.html?style=./suggestion.styl';
+import Vue from 'vue';
+import { Component, Emit, Prop } from 'vue-property-decorator';
+
+export class TagSuggester {
```

Comment:
I kind of think we should just put this in the component like we had before so that vue can watch it all easier and update appropriately.